src/components/_modal/pages/references/LinksList.js

```diff
@@ -0,0 +1,48 @@
+import Translate from "@/components/base/translate";
+import Link from "@/components/base/link";
+import Text from "@/components/base/text/Text";
+import getConfig from "next/config";
+
+// eslint-disable-next-line css-modules/no-unused-class
+import styles from "./References.module.css";
+
+/**
+ * TODO hook up with @jopr s implementering
+ * @param {String} pid
+ * @returns
+ */
+const onlinelinks = (pid) => {
+  const APP_URL =
+    getConfig()?.publicRuntimeConfig?.app?.url || "http://localhost:3000";
+
+  return {
```

Comment:
skal nok lige tilpasse, Der skal vidst bare ændre pid -> pids i url'er (og modtage pids som array istedet for pid) - så tror jeg den er good to go